doc/locale/ja/LC_MESSAGES/reference/commands/select.po

```diff
@@ -3185,6 +3186,41 @@ msgstr ""
 "ません。この場合は ``_key`` を使ってください。これは、 :ref:`select-"
 "drilldown-output-columns` と同じルールです。"
 
+msgid "Specify ``${LABLE}`` of other ``drilldowns`` or ``slices``."
+msgstr "他の ``drilldowns`` または ``slices`` の ``${LABLE}`` を指定します。"
+
+msgid ""
+"You can drilldown the result of specified ``${LABLE}``. It means that this "
+"parameter enables a nested drilldown."
+msgstr ""
+"指定した ``${LABLE}`` の結果をドリルダウンします。つまり、このパラメータを使"
+"うと多段ドリルダウンができます。"
+
+msgid ""
+"Here is an example to execute the nested drilldown by tag and then by "
```

Comment:
These `tag` and  `category`  in this sentence are different from the labeling?  If they meant the label, I think it would be better to keep `` mark. 

Also it might be more thoughtful to say following.
If I understand the nested drilldown correctly, I believe final result is sorted with 2 step of drilldown and I have broken down the steps bit more.

 ```suggestion
"Here is an example to execute the nested drilldown. The final result takes first drilldown by tag and then 2nd drilldown by category against first result."
```

Reply:
I may need to ask what is the nested drilldown first  in Japanese....

Reply:
多段ドリルダウンの挙動についてはご認識のとおりです。

Reply:
``で括っている時は、例示したクエリで作ったカラムについて言及しているのですが、本文中ではこの時点ではまだ例を示していないので、括らないほうがよいのではないかと悩んだ末にこうなっていました。

``でくくるようにしようと思います。